tests/affiliatedcertification/tests/affiliated_certification_container.go

```diff
@@ -17,22 +23,94 @@ var _ = Describe("Affiliated-certification container certification,", func() {
 
 	// 46562
 	It("one container to test, container is certified", func() {
-		Skip("Under development")
+		By("Add container information to tnf_config.yml")
+		err := globalhelper.DefineTnfConfig(
+			[]string{netparameters.TestNetworkingNameSpace},
+			[]string{netparameters.TestPodLabel},
+			[]string{affiliatedcertparameters.CertifiedContainer1})
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
Let's add more logs. Please add optionalDescritpion for every expect call.
For example: Expect(err).ToNot(HaveOccurred(), "Error to define tnf config file")